Fix `PyUnicodeDecodeError_Create` failing unconditionally with a TypeError on PyPy.
